CreeDictionary/DatabaseManager/xml_importer.py

```diff
@@ -520,22 +513,43 @@ def import_xmls(dir_name: Path, multi_processing: int = 1, verbose=True):
                         keyword_counter += 1
 
                 # now we create definition for all (possibly inflected) entries in xml that are forms of this lemma.
-                d_s_dicts = xml_lemma_to_definition_src_dicts.get(generated_wordform)
+                # d_s_dicts = xml_lemma_to_definition_src_dicts.get(generated_wordform)
+
+                # try to match our generated wordform to entries in the xml file,
+                # in order to get its definition from the entries
+                d_s_dicts: List[dict] = []
+
+                # first get homographic entries from the xml file
+                pos_lc_tuples_in_xml = xml_lemma_to_pos_lc.get(generated_wordform)
+
+                # The case when we do have holographic entries in xml,
```

Comment:
```suggestion
                # The case when we do have homographic entries in xml,
```

![well holo everyone](https://media1.tenor.com/images/14856c9146633ef1c2554f03cf0f6d91/tenor.gif?itemid=15711177)